.github/workflows/rust.yml

```diff
@@ -53,6 +72,16 @@ jobs:
         # the binary will be used by the next cargo test step
         run: cargo build
       - name: Run cargo tests
-        run: cargo test --all-targets --all-features
+        env:
+          GITHUB_ACTIONS_BASH_CONTAINER: 1
+        run: |
+          docker run --volume $PWD:/root --network host --detach --name bash bash:${{ matrix.bash_version }} sleep infinity
+          # coreutils includes mktemp
+          docker exec bash apk add coreutils curl
+          if [ "${{ matrix.sed }}" == "GNU" ]; then
+            docker exec bash apk add sed
+          fi
+          cargo test --all-targets --all-features
```

Comment:
Do I understand correctly that this will run validation and stdlib tests on all versions of Bash in the matrix above?

Reply:
yes